pxr/usd/usdLux/lightAPI.h

```diff
@@ -359,10 +420,120 @@ class UsdLuxLightAPI : public UsdAPISchemaBase
     // --------------------------------------------------------------------- //
     // NORMALIZE 
     // --------------------------------------------------------------------- //
-    /// Normalizes power by the surface area of the light.
+    /// Normalizes the emission such that the power of the light
+    /// remains constant while altering the size of the light, by dividing the
+    /// luminance by the world-space surface area of the light.
+    /// 
     /// This makes it easier to independently adjust the power and shape
-    /// of the light, by causing the power to not vary with the area or
-    /// angular size of the light.
+    /// of the light, by causing the total illumination provided by a light to
+    /// not vary with the area or angular size of the light.
+    /// 
+    /// More precisely, this means that the luminance of the light will be
+    /// divided by a factor representing the "size" of the light:
+    /// 
+    /// 
+    /// <center><b>
+    /// L<sub>Scalar</sub> = L<sub>Scalar</sub> / sizeFactor
+    /// </b></center>
+    /// 
+    /// 
+    /// ...where `sizeFactor` = 1 if `normalize` is off, and is calculated
+    /// depending on the family of the light as described below if `normalize`
+    /// is on.
+    /// 
+    /// ### DomeLight / PortalLight:
+    /// 
+    /// For a dome light (and it's henchman, the PortalLight), this attribute is
+    /// ignored:
+    /// 
+    /// <center><b>
+    /// sizeFactor<sub>dome</sub> = 1
+    /// </b></center>
+    /// 
+    /// ### Area Lights:
+    /// 
+    /// For an area light, the `sizeFactor` is the surface area (in world
+    /// space) of the shape of the light, including any scaling applied to the
+    /// light by its transform stack. This includes the boundable light types
+    /// which can have a calculable surface area, such as:
```

Comment:
we should be more normative. Instead of "can have" we should say "does have". If there is an exception we should call it out.

Reply:
Changed to "light types which have":

https://github.com/PixarAnimationStudios/OpenUSD/pull/3182/commits/a1d7d8328149bf6067832d65c79f003fcfc262fa#diff-11634bb5438749e3c529c9f44011d464e79bdb1352c0b7fc7c9408bece4415e4R457